redfish-core/lib/memory.hpp

```diff
@@ -607,13 +615,6 @@ inline void getDimmDataByService(std::shared_ptr<bmcweb::AsyncResp> asyncResp,
                                  const std::string& service,
                                  const std::string& objPath)
 {
-    if constexpr (bmcwebEnableHealthPopulate)
```

Comment:
We don't have this enabled so this is a no-op

Reply:
Maybe we will enable this for https://github.com/ibm-openbmc/bmcweb/pull/836/files ?